integration_tests/.env/bigquery.env

```diff
@@ -1 +1,2 @@
-GCLOUD_SERVICE_KEY_PATH=
\ No newline at end of file
+BIGQUERY_SERVICE_KEY_PATH=
+BIGQUERY_TEST_DATABASE=
```

Comment:
I don't think there's anything preventing folks from accidentally committing secrets that they write to locations like `integration_tests/.env/bigquery.env`